doc/start/documenting-ceph.rst

```diff
@@ -60,32 +60,33 @@ instead of compiling program source. This sequence (the sequence of building the
 Get the Source
 --------------
 
-Ceph documentation lives in the Ceph repository right alongside the Ceph source
-code under the ``ceph/doc`` directory. For details on github and Ceph,
-see :ref:`Get Involved`.
+The source of the Ceph documentation is a colletion of ReStructured Text files
+that are in the Ceph repository in the ``ceph/doc`` directory.  For details
```

Comment:
within the Ceph git repository under the 

Also, here you add a second space after the period, in other PRs I've been admonished to -- contrary to childhood inculcation -- only use one.

Reply:
I think that you've just helped me codify a gossamer distinction: "in a directory" is correct when the file referred to is located directly in the directory specified, and "under a directory" is correct when what is referred to is a structure (in this case, a collection of documents that ramify like a tree) that is located in the directory specified (but of course, it is not a mere single file or single directory that is referred to).

I'll take out the extra space.